src/app/(app)/(home)/search-filters/search-input.tsx

```
@@ -0,0 +1,39 @@
+
+import { useState } from 'react';
+import { Button } from '@/components/ui/button';
+import { Input } from '@/components/ui/input';
+import { ListFilterIcon, SearchIcon } from 'lucide-react';
+
+import { CategoriesSidebar } from './categoriesSidebar';
+interface Props {
+  disabled?: boolean;
+}
+
+export const SearchInput = ({ data, disabled }: Props) => {
```

ESLint

[error] 'data' is defined but never used.

(@typescript-eslint/no-unused-vars)